modules/cef/filterx-func-parse-leef.c

```diff
@@ -87,6 +87,21 @@ _is_delmiter_empty(gchar delimiter)
   return delimiter == 0;
 }
 
+static gboolean
+_fallback_to_parse_extensions(EventParserContext *ctx, const gchar *input, gint input_len, FilterXObject **result,
+                              GError **error, gpointer user_data)
+{
+  if (!csv_scanner_append_rest(ctx->csv_scanner))
```

Comment:
Shouldn't `error` be set?

Reply:
Pushed a commit, we can drop it if you don't like it.